From 349d947a8d1665295f37f842dfe228afca536621 Mon Sep 17 00:00:00 2001
From: Jian Li <jian.li@freescale.com>
Date: Thu, 14 Mar 2013 13:25:27 +0800
Subject: [PATCH 174/602] ENGR00133115 add lib_omx_ac3toiec937_arm11_elinux to
 usr build

add lib_omx_ac3toiec937_arm11_elinux to usr build

Signed-off-by: Jian Li <jian.li@freescale.com>
---
 imx5x/imx5x.mk | 1 +
 imx6/imx6.mk   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/imx5x/imx5x.mk b/imx5x/imx5x.mk
index 2a052c2..3f311ae 100644
--- a/imx5x/imx5x.mk
+++ b/imx5x/imx5x.mk
@@ -235,6 +235,7 @@ omx_libs := \
 	lib_mp3_parser_arm11_elinux.3.0 \
 	lib_flac_parser_arm11_elinux.3.0 \
 	lib_wav_parser_arm11_elinux.3.0 \
+	lib_omx_ac3toiec937_arm11_elinux \
 
 
 # Omx excluded libs
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 811c55c..c3d363b 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -236,6 +236,7 @@ omx_libs := \
 	lib_mp3_parser_arm11_elinux.3.0 \
 	lib_flac_parser_arm11_elinux.3.0 \
 	lib_wav_parser_arm11_elinux.3.0 \
+	lib_omx_ac3toiec937_arm11_elinux \
 
 
 
-- 
1.8.0

